requirements.txt

```diff
@@ -29,3 +29,5 @@ redis==2.9.0
 nydus==0.10.6
 Cerberus==0.5
 matplotlib
+git+https://github.com/asl2/PyZ3950.git
```

Comment:
PyZ3950 is on pypi and latest version corresponds to the github master. So you can simply put `PyZ3950==2.04`